pkgs/development/libraries/zlib/default.nix

```diff
@@ -78,4 +78,8 @@ stdenv.mkDerivation rec {
     license = licenses.zlib;
     platforms = platforms.all;
   };
-}
+} // stdenv.lib.optionalAttrs (stdenv.hostPlatform != stdenv.buildPlatform) {
+  preConfigure = ''
```

Comment:
This is pretty weird, is it wrong to define preConfigure always with an optional value?

Reply:
That would be a mass rebuild.

Reply:
I'll do a staging PR after.